composer.json

```diff
@@ -18,12 +18,12 @@
     "composer/composer": "^2",
     "composer/semver": "^3",
     "consolidation/comments": "^1.0.2",
-    "consolidation/filter-via-dot-access-data": "^1.0",
+    "consolidation/filter-via-dot-access-data": "^2.0",
     "consolidation/output-formatters": "^4",
     "consolidation/robo": "^3",
     "consolidation/site-alias": "^3.1",
     "czproject/git-php": "^4.0",
-    "guzzlehttp/guzzle": "^6.2",
+    "guzzlehttp/guzzle": "^6.2 || ^7.0",
```

Comment:
We don't really support using Terminus as a library; folks who do that will have to adapt to our dependencies. You can drop the `||` and just keep the highest constraint.